src/sentry/api/endpoints/data_export.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+from sentry import features
+from sentry.api.bases.incident import IncidentPermission
+from sentry.api.base import Endpoint
+from django.utils import timezone
+
+
+from sentry.tasks.data_export import create_record
+
+from sentry.models import ExportedData, Organization
+
+
+class DataExportEndpoint(Endpoint):
+    permission_classes = (IncidentPermission,)
+
+    def get(self, *args, **kwargs):
+        """
+        Retrieve information about the temporary file record.
+        Used to populate page emailed to the user.
+        """
+        organization = Organization.objects.get(slug=kwargs["organization_slug"])
+        if not features.has("organizations:data-export", organization):
+            return self.respond(status=404)
```

Comment:
Feature flag to prevent access to this API endpoint, to test, you have to turn on `organizations:data-export` in `SENTRY_FEATURES`